tools/ci/platforms/ubuntu.sh

```diff
@@ -0,0 +1,543 @@
+#!/usr/bin/env bash
+############################################################################
+# tools/ci/platforms/ubuntu.sh
```

Comment:
Just for my understanding: what is the delta between `tools/ci/platforms/linux.sh` and `tools/ci/platforms/ubuntu.sh`?

Reply:
It is currently the same. This is a base for Linux distros that do not have the /etc/os-release file.

Reply:
Maybe let's rename `linux.sh` to `debian.sh` as it early relies on `apt-get` and I think will not work for Fedora?

Reply:
Recent Debian and Fedora distros have the /etc/os-release file and are already present in the find_platform function.

[function -> find_platform](https://github.com/apache/nuttx/blob/81d163710f2bcedc196d6890bf730fd25e92abe3/tools/ci/cibuild.sh#L46)

All that is missing is to create the script file (debian.sh and fedora.sh) with with the correct tool packages and toolchains.
If the /etc/os-release file is missing you search platform via "uname -s" (darwin, cygwin etc.).  So linux.sh script is for all those distros dated or without  /etc/os-release file that you don't know package manager. Maybe it is better to delete from linux.sh the apt-get packages and leave only the toolchains downloads.
What do you think ?